### PR TITLE
Mar 2510. Added adaptability to banner 

### PR DESCRIPTION
**Pull request by Antoneo-MD**
task: 
https://maddevs.atlassian.net/jira/software/c/projects/MAR/boards/47?modal=detail&selectedIssue=MAR-2510&assignee=61acf073977c5b0072556985

description:
Added adaptability to banner 

tests:
image

### CODE REVIEW

File: client/components/core/banners/MadCommunityBanner.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import UIButton from '@/components/shared/UIButton'
+
+export default {
+  name: 'MadCommunityBanner',
+  components: {
+    UIButton,
+  },
+
+  data() {
+  },
```
Comment (TangoPJ) [inline client/components/core/banners/MadCommunityBanner.vue:48]: Удали пустую data(){}

File: client/components/core/banners/MadCommunityBanner.vue [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+<script>
+import UIButton from '@/components/shared/UIButton'
+
+export default {
+  name: 'MadCommunityBanner',
+  components: {
+    UIButton,
+  },
+
+  data() {
```
Comment (TangoPJ) [inline client/components/core/banners/MadCommunityBanner.vue:47]: Удали пустую data() {}

**Review by TangoPJ — APPROVED** [review]

**Review by Ernest2077 — APPROVED** [review]
Approved

**Review by oh-cactus — APPROVED** [review]

**Comment by Antoneo-MD** [comment]
add MadCommunityBanner to Blog page

**Pull request merged**